packages/pipeline-editor/src/index.ts

```diff
@@ -170,19 +223,42 @@ const extension: JupyterFrontEndPlugin<void> = {
       category: 'Elyra'
     });
 
+    const schema = await PipelineService.getRuntimesSchema().catch(error =>
+      RequestErrors.serverError(error)
+    );
```

Comment:
this should probably be a try/catch if we are  using async/await

Reply:
I meant you should be able to do something like this: 
```
let schema 
try {
  schema = await PipelineService.getRuntimesSchema()
} catch (e) {
  RequestErrors.serverError(e)
}
```

Reply:
Ohh ok sounds good I'll add that

Reply:
So the rest of the code that uses the `schema` variable will need to be in that try as well though, right? Also what is the benefit of using try/catch over the await/catch?

Reply:
Schema will just be undefined if it throws an error 

.catch() is just using the promise syntax, there’s nothing wrong with it, but we should stick to async/await or the promise syntax and not mix and match

Also, typescript wise, I think this would be consider schema as defined instead of potentially undefined:
```
const schema = await getSchema().catch()
```

Reply:
Oh ok thanks for explaining - is the way it's implemented currently (see https://github.com/elyra-ai/elyra/pull/1591/files#diff-2280d00b71af312d9eecec1377484159361d5145193a35ef246a7fa9a37abaf1R234) using a `then` with resolve/reject handlers and all logic that uses the `schema` variable equivalent though?